dotCMS/src/main/java/com/dotcms/rest/api/v1/site/SiteResource.java

```diff
@@ -897,6 +899,9 @@ public Response updateSite(@Context final HttpServletRequest httpServletRequest,
             throw new NotFoundException("Site: " + siteIdentifier + " does not exists");
         }
 
+        if (null != site.getMap().get(Contentlet.NULL_PROPERTIES)) {
+            Set.class.cast(site.getMap().get(Contentlet.NULL_PROPERTIES)).clear();
+        }
```

Comment:
There's a Contentlet.cleanUp public Method that accomplishes what you need.

Reply:
Maybe that clean-up logic should take place within the helper .. don't you think?

Reply:
thanks change done